sdg/edges.py

```diff
@@ -46,6 +44,9 @@ def x_without_y(x, y):
         x (pandas Series): Left hand column
         y (pandas Series): Right hand column
     """
+    # returns true if there are any cases where an element in
```

Comment:
These comments should go in the doc string
```suggestion
    """
     Args:
        x (pandas Series): Left hand column
        y (pandas Series): Right hand column
    
    Returns:
        bool: True if there are any cases where an element in the y column is 
                 empty (is a nan) and the corresponding element in the x column
                 is not empty
    """
```